README.md

```diff
@@ -225,10 +225,11 @@ GET /status                              - returns middleware status
 
 ## OpenAPI specs
 
-Under the `/swagger` path there are two files providing a swagger specificiation of the endpoints for both api versions:
+There are two files providing a swagger specificiation of the endpoints for respective api versions:
+
+- https://mainnet.aeternity.io/mdw/swagger/swagger_v1.yaml
+- https://mainnet.aeternity.io/mdw/swagger/swagger_v2.yaml
```

Comment:
thats incorrect

Reply:
What is incorrect? Please try with .art for v2.

Reply:
It is not deplyed. Please try with .art for v2.